apps/service-portal/src/store/modules.ts

```diff
@@ -45,11 +46,13 @@ export type ModuleKeys =
   | 'licenses'
   | 'petitions'
   | 'vehicles'
+  | 'delegations'
 
 export const featureFlaggedModules: ModuleKeys[] = [
   'documentProvider',
   'icelandicNamesRegistry',
   'petitions',
+  'delegations',
```

Comment:
Fyi, we recently encountered an issue in production when cleaning up this kind of magic feature flag which is shared with the API. For this reason I would like to change this `featureFlaggedModules` functionality to depend on feature flags exported from `@island.is/feature-flags`.

Reply:
Discussed locally. I will merge wait with the merge until the feature flagged modules functionality is merged